app/controllers/maintainers_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class MaintainersController < ApplicationController
+  before_action :load_project
```

Comment:
_disclaimer: nem fogom csak azért visszadobni, meg gondold végig, hogy neked is segítene-e ez a kódstílus. nem kötelező így csinálni, csak egy ötlet_

Én a `before_action` és az első action közé tennék egy üres sort.